aten/src/ATen/native/cpu/layer_norm_kernel.cpp

```diff
@@ -282,7 +282,7 @@ void LayerNormBackwardKernelImpl(
   AT_DISPATCH_FLOATING_TYPES(
       X.scalar_type(), "LayerNormBackwardKernelImpl", [&]() {
         LayerNormBackwardKernelImplInternal<scalar_t>(
-            dY, X, mean, rstd, gamma, M, N, dX, dgamma, dbeta);
+            dY.contiguous(), X, mean, rstd, gamma, M, N, dX, dgamma, dbeta);
```

Comment:
Why is this needed now?

Reply:
This is because of autodiff from JIT.

Back then we have autograd calling contiguous on grads[0] when it is not contiguous. But that's not visible to autodiff and causes breakage. We want to keep things consistent, so we are now calling contiguous here and removed the corresponding part in autodiff (https://github.com/pytorch/pytorch/pull/50467/files#diff-e4c2f99a2404e98c3586e07425da73008f36b1bada790648a7297af141d37f8cR975)

Reply:
Ho ok, It was hard to see what was removed from that formula :D